src/utils/remove-path-prefix-from-url.js

```diff
@@ -0,0 +1 @@
+export const removePathPrefixFromUrl = url => url.replace(__PATH_PREFIX__, '');
```

Comment:
Simple logic but extracted into a reusable function.

This allows us to remove the path prefix we generate for individual builds on a string (i.e. on this branch's staging link, the path prefix is `/master/devhub/jordanstapinski/DEVHUB-362/`). We don't have to do this to solve this issue, but doing so allows others to better preview what the canonical URL for a page will be once in production.